lib/improver/tests/utilities/test_load.py

```diff
@@ -225,6 +225,11 @@ def test_ordering_for_realization_threshold_percentile_over_coordinate(
         self.assertArrayAlmostEqual(result.coord_dims("latitude")[0], 4)
         self.assertArrayAlmostEqual(result.coord_dims("longitude")[0], 5)
 
+    def test_attributes(self):
+        """Test that metadata attributes are successfully striped out."""
+        result = load_cube(self.filepath)
+        self.assertNotIn('bald__isPrefixedBy', result.attributes.keys())
+
```

Comment:
Can we add a new unit test to check that the "prefixes" cube is not loaded.

Reply:
Good point! - unittest now added that checks there is no prefix cube after load. Also corrected typo and removed spp__form of difference line as per comments.